rs/sns/governance/src/governance.rs

```diff
@@ -5174,6 +5131,78 @@ impl Governance {
         self.proto.metrics.replace(metrics);
     }
 
+    /// Computes the total potential voting power of the governance canister and electoral roll.
+    fn governance_total_potential_voting_power(
+        &self,
+    ) -> Result<(u64, BTreeMap<String, Ballot>), String> {
```

Comment:
Note to reviewer: 

This function's implementation is factored out from `make_proposal`.

Reply:
I think it's primary function is making ballots, so maybe this function name kind of forgets the most important thing it does?

I think we have a good name for a similar function in NNS: compute_ballots_for_new_proposal

That function does a few other things, but it returns the total_potential_voting_power